src/patterns/names/index.md.njk

```diff
@@ -49,10 +51,18 @@ If users are from outside the UK, use the labels:
 - ‘Given names’
 - ‘Family name’
 
-Make middle names optional.
-
 Make it clear whether you need someone’s common name, or their name as it's written on official documents such as a passport or driving licence.
 
+#### Middle names
+
+Only ask for middle names if your service requires them. 
+
+Use the label:
+
+- ’Middle names‘
+
+Make middle names optional as not all users have middle names. They do not need `(optional)` in the label as users should provide them if they have one - and will skip the field if they do not have one.
```

Comment:
Make sure middle names are optional, as not everyone has them.

The label should not include `(optional)`. Users will enter their middle names if they have them, and skip the field if they do not.